internal/app/machined/pkg/system/services/osd.go

```diff
@@ -141,13 +116,7 @@ func (o *OSD) Runner(config runtime.Configurator) (runner.Runner, error) {
 // HealthFunc implements the HealthcheckedService interface
 func (o *OSD) HealthFunc(runtime.Configurator) health.Check {
 	return func(ctx context.Context) error {
-		var d net.Dialer
```

Comment:
We should change this to use the local socket. I would hate to lose the health check.